apps/src/lib/node/ledger/shell/prepare_proposal.rs

```diff
@@ -12,7 +16,7 @@ use namada::types::transaction::{AffineCurve, DecryptedTx, EllipticCurve};
 
 use super::super::*;
 #[allow(unused_imports)]
-use super::block_space_alloc;
+pub use super::block_space_alloc;
```

Comment:
why `pub use` here? It is pub in the super mod

Reply:
Without the `pub` here I could not build the docs because of some unresolved links

Reply:
ah I see - I think it would be better to change the docs to use `super::block_space_alloc` and remove the unused import here and in ` apps/src/lib/node/ledger/shell/block_space_alloc/states.rs` remove the `prepare_proposal::` from docs that refer to `block_space_alloc`